README.md

```diff
@@ -90,44 +90,44 @@ $ curl -sfLO https://raw.githubusercontent.com/intelsdi-x/snap-plugin-collector-
 Ensure to provide your Keystone instance address and credentials.
 
 Ensure [Snap daemon is running](https://github.com/intelsdi-x/snap#running-snap) with provided configuration file:
-* command line: `snapd -l 1 -t 0 --config cfg.json&`
+* command line: `snapteld -l 1 -t 0 --config cfg.json&`
```

Comment:
In other plugins there is no trailing `&` for this command. Also we prefer to give example using `systemctl` or `service` (see other plugins). Personally I did example in nova collector with downloading example global config to default config location.